src/library/scala/math/package.scala

```diff
@@ -315,13 +356,83 @@ package object math {
   /** Returns the size of an ulp of the given `Double` value.
     * @group ulp
     */
-  def ulp(x: Double): Double = java.lang.Math.ulp(x)
+  def ulp(d: Double): Double = java.lang.Math.ulp(d)
```

Comment:
I agree with you, and at first in Java docs you can see - https://docs.oracle.com/javase/8/docs/api/java/lang/Math.html#ulp-double-

ulp(double d)/ulp(float f)

and second, i think, if we use convention, that we base naming on Java docs/Java source code, we need to do this in all places in file, not just in new bindings.